src/include/common/constants.h

```diff
@@ -150,6 +150,14 @@ struct CopyConstants {
     static constexpr const char* INT_CSV_PARSING_OPTIONS[] = {"SKIP", "SAMPLE_SIZE"};
     static constexpr uint64_t DEFAULT_CSV_SKIP_NUM = 0;
     static constexpr uint64_t DEFAULT_CSV_TYPE_DEDUCTION_SAMPLE_SIZE = 256;
+
+    // metadata columns used to populate CSV warnings
+    static constexpr std::array WARNING_METADATA_COLUMN_NAMES = {"startByteOffset", "endByteOffset",
+        "fileIdx", "blockIdx", "rowOffsetInBlock"};
```

Comment:
I feel like there is no simple way to calculate `blockIdx` for `SerialCSVReader` so I left `blockIdx` in here (since we already minimize performance cost if `IGNORE_ERRORS=false`, this performance hit will only happen if users are expecting warnings). Let me know if there is a simple way and I can make this minor optimization.

Reply:
Added todo to `WarningSourceData` struct